server/block/smelter.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 )
 
-// smelter is a struct that may be embedded by blocks that can smelt blocks and items, such as blast furnaces, furnaces,
+// smelter is a struct that may be embedded by blocks that can smelt blocks and items, such as blast smelters, smelters,
```

Comment:
This looks like an unintended doc change due to a refactor. Check other docs in this file too!